tests/integration_test/cloud_functions/cloud_functions_e2e_test.dart

```diff
@@ -120,6 +122,20 @@ void main() {
           expect(result.data, isA<Map<String, dynamic>>());
         },
       );
+
+      test('can be called using an Uri', () async {
+        final localhostMapped =
+            kIsWeb || !Platform.isAndroid ? 'localhost' : '10.0.2.2';
```

Comment:
Is this needed: `kIsWeb ||`? If it is not android, it will hit `localhost`, right?

Reply:
Platform is not available on Web, so you have to use kIsWeb to remove the "Platform" when on Web during compilation :)

Reply:
ah, yes, I had a vague memory of something like that 👍